### PR TITLE
Replace Goutte by Guzzle and adding testing suite

### PR DESCRIPTION
**Pull request by peter279k**
I revert the phpunit.xml.dist settings and squash commits to my first origin commit.

### CODE REVIEW

**Comment by Th3Mouk** [comment]
Why opening so much PRs ?

**Comment by peter279k** [comment]
Uh...the #4 is that I cannot revert the phpunit.xml.dist and push the commits again.
So I create the new pull request #5.

**Comment by Th3Mouk** [comment]
You can commit the changes on same branch, and after squash them. This will be revert changes.

File: phpunit.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<phpunit
+    colors="true"
+    bootstrap="./vendor/autoload.php"
+    type="coverage-text"
+    target="tests/"
+    charset="UTF-8">
+
+    <testsuites>
+        <testsuite name="testing">
+      	    <directory suffix=".php">tests/</directory>
```
Comment (Th3Mouk) [inline phpunit.xml]: ./tests/

File: phpunit.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        </testsuite>
+    </testsuites>
+
+    <filter>
+        <whitelist>
+            <directory suffix=".php">src/</directory>
```
Comment (Th3Mouk) [inline phpunit.xml]: ./src/

File: phpunit.xml.dist [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
+<phpunit
+    backupGlobals="false"
+    backupStaticAttributes="false"
+    convertErrorsToExceptions="true"
+    convertNoticesToExceptions="true"
+    convertWarningsToExceptions="true"
+    processIsolation="false"
+    colors="true"
+    bootstrap="./vendor/autoload.php"
+    type="coverage-text"
```
Comment (Th3Mouk) [inline phpunit.xml.dist]: Can u help me on the utility of this ?

File: phpunit.xml.dist [hunk 4]
```diff
@@ -1,27 +1,32 @@
...
+    target="tests/"
```
Comment (Th3Mouk) [inline phpunit.xml.dist]: And this ?

Comment (peter279k) [inline phpunit.xml.dist]: Do you mean the type=```"coverage-text"``` and ``` target="tests/"``` are not used ? They have to be removed?

**Pull request merged**